.circleci/config.yml

```diff
@@ -1919,30 +1919,6 @@ workflows:
           requires:
             - setup
             - pytorch_windows_vs2019_py36_cuda10.1_build
-      - pytorch_linux_build:
```

Comment:
is this from editing the `.circleci/config.yml` directly?

You'll need to remove it in the generator for it in `.circleci/cimodel/data/pytorch_build_data.py`

Possibly even just a bump from `3.5` -> `3.6` would fare better instead of just a straight removal.

Reply:
Why do we need vanilla py3.6? py3.6-gcc7 would be exactly the same, isn't it?